restorer_test.go

```diff
@@ -309,20 +320,32 @@ func testRestorerBuilder(buildpackAPI, platformAPI string) func(t *testing.T, wh
         }
     ]
 }
-`, cacheFalseLayerSHA, cacheLaunchLayerSHA, cacheOnlyLayerSHA, noGroupLayerSHA, escapedLayerSHA)
+`, bomLayerSHA, cacheFalseLayerSHA, cacheLaunchLayerSHA, cacheOnlyLayerSHA, noGroupLayerSHA, escapedLayerSHA)
 
 					err = ioutil.WriteFile(
 						filepath.Join(cacheDir, "committed", "io.buildpacks.lifecycle.cache.metadata"),
 						[]byte(contents),
 						0600,
 					)
 					h.AssertNil(t, err)
+
+					h.Mkdir(t, filepath.Join(layersDir, "bom-buildpack.id"))
 				})
 
 				it.After(func() {
 					h.AssertNil(t, os.RemoveAll(tarTempDir))
 				})
 
+				it("restores any BOM files to the correct locations", func() {
+					h.SkipIf(t, api.MustParse(platformAPI).LessThan("0.8"), "PLATFORM API <0.8 does not restore sBOM")
```

Comment:
```suggestion
					h.SkipIf(t, api.MustParse(platformAPI).LessThan("0.8"), "Platform API < 0.8 does not restore sBOM")
```
This will make my OCD happy...
```suggestion
					h.SkipIf(t, api.MustParse(platformAPI).LessThan("0.8"), "PLATFORM API <0.8 does not restore sBOM")
```

Reply:
_Fixed in latest commit._